test/scripts/gateway/install-gateway.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+
```

Comment:
I general would like to see us standardize on fail fast with bash scripts. WDYT?

```suggestion
set -eo pipefail
```

Reply:
sure, added